rmf_gazebo_plugins/src/teleport.cpp

```diff
@@ -93,6 +96,13 @@ class TeleportPlugin : public gazebo::ModelPlugin
       RCLCPP_INFO(_node->get_logger(),  "Unloading object");
       _model->SetWorldPose(_unload_pose);
       _object_loaded = false;
+
+      // Hard coded: Leave object at goal location for 2.0 second, then
+      // teleport
+      // it back to initial ( pre pickup  ) location
+      rclcpp::sleep_for(std::chrono::nanoseconds(2000000000));
```

Comment:
To improve readability
```suggestion
      rclcpp::sleep_for(std::chrono::seconds(2));
```